csss-site/src/events/templates/events/frosh/frosh_week.html

```diff
@@ -1,20 +1,20 @@
-{% extends "csss/new_header.html" %}
-
-{% load static %}
-{% load staticfiles %}
-
-{% block content %}
-    <h1 class="title">Frosh Week</h1>'
-    <p class="lead">FroshWeek is the ultimate introductory experience for Computing Science students at Simon Fraser University.</p>
-    <p class="lead">Organized by us, the CSSS, in partnership with the School of Computing Science, it is a week of fun-filled activities designed to orient, entertain and educate new university students with a Computing Science twist.</p>
-    <p class="lead">Held during the first week of the Fall semester at SFU <b>(Sept. 4 - 8)</b>, FroshWeek is organized by students, for students. Incoming students receive: an in-depth introduction to the facilities available to them as Computing Science students; many entertaining on- and off-campus challenges and diversions to motivate, amuse, and connect them with other students; valuable advice from their elders with respect to courses and the university experience in general; and some free food and tools to make the transition to university much smoother.</p>
-{#    <p class="lead">Check out <a href="{{ URL_ROOT }}events/frosh/2019">The Official Frosh Week Site</a> for more info, and to register!</p>#}
-    <p class="lead">Check out <a href="{%  static 'events_static/frosh/Under-Construction1.jpg' %}">The Official Frosh Week Site</a> for more info, and to register!</p>
-    <h1 class="title">The RAMing Ceremony</h1><hr>
-    <p class="lead">Our most hallowed of rituals. It doesn't hurt, we promise.</p>
-    <h1 class="title">Pac Macro</h1>
-    <hr><p class="lead">The Frosh Week classic: become Pacman himself among the streets of downtown Vancouver. Pacman and the ghosts chase each other amidst the crowds and traffic, all the while being directed by their team mates, stationed at Harbour Centre. There, they have a live overview of the positions of all players, projected on a large screen, and managed by custom CSSS software.</p>
-    <h1 class="title">Midnight Madness</h1>
-    <hr>
-    <p class="lead">An all-night event in the ASB, where the four Applied Science schools (Comp Sci, Engineering, Software Systems, and Mechtronics) go head-to-head in various challenges, until they arrive at the conclusion that everyone knew already: Comp Sci is the best.</p>
+{% extends "csss/new_header.html" %}

```

Comment:
this file should not have so much changed. do a re-fresh from csss master and then just change the relevant lines